src/ray/gcs/gcs_server/gcs_health_check_manager.cc

```diff
@@ -14,14 +14,18 @@
 
 #include "ray/gcs/gcs_server/gcs_health_check_manager.h"
 
+#include <memory>
+#include <string>
 #include <string_view>
+#include <utility>
+#include <vector>
 
 #include "ray/stats/metric.h"
 
 DEFINE_stats(health_check_rpc_latency_ms,
              "Latency of rpc request for health check.",
              (),
-             ({1, 10, 100, 1000, 10000}, ),
+             ({1, 10, 100, 1000, 10000}, ), // NOLINT
```

Comment:
```sh
src/ray/gcs/gcs_server/gcs_health_check_manager.cc:24:  Extra space before )  [whitespace/parens] [2]
```
Sounds reasonable to me, maybe we just remove the space? Curious does it work if we make it
```
({1, 10, 100, 1000, 10000}),
```

Reply:
Fixed.